lib/srv/db/sqlserver/kinit/kinit.go

```diff
@@ -361,3 +338,10 @@ func (k *CommandLineInitializer) WriteKRB5Config(path string) error {
 
 	return trace.Wrap(os.WriteFile(path, []byte(s), 0644))
 }
+
+// buildAnchorsFileContents generates the contents of the anchors file (pkinit).
+// The file must contain the Teleport DB CA and the KDB/LDAP CA, otherwise the
+// connections will fail.
+func (k *CommandLineInitializer) buildAnchorsFileContents(caBytes []byte) []byte {
+	return append(caBytes, []byte(k.ldapCertificatePEM)...)
```

Comment:
do we need a new line?

Reply:
If we are sure it has a new line then it's ok. If we are not sure, an extra newline wouldn't hurt.